teamcode.html

```diff
@@ -58,11 +58,7 @@ <h2>TeamCode</h2>
         office hours, and GitHub. Code will be checked into GitHub and reviewed
         by other team members who can then merge work into the main repository.
       </p>
-      <p>The TeamCode team for Fall 2019 consists of:</p>
-      <ul>
-        <li><b>Brenton Strine</b> (Advisor)</li>
-        <li><b>Mustafa Alsaeedi</b> (Devloper)</li>
-      </ul>
+
       <img src="media/collaboration.jpg">
```

Comment:
I think you need to come in here and manually fix this. If you look at the changes above, this PR will currently delete the list of TeamCode members. 

To manually fix this, you'll need to do the following: 
1. From **GitHub Desktop** fetch or pull from origin. (_This gets the code from GitHub to your computer.)
2. Open `teamcode.html` in Atom
3. Find the section above the `<img src="media/collaboration.jpg"> image`. Looks like it's line 63.
4. Add back both the old code and your new code. Should look like this:
```
      <p>The TeamCode team for Fall 2019 consists of:</p>	
       <ul>	
        <li><b>Brenton Strine</b> (Advisor)</li>	
        <li><b>Mustafa Alsaeedi</b> (Devloper)</li>	
        <li>Sinatayehu Mekonnen (Developer)</li>
      </ul>```
5. Commit
6. Push to origin (This puts the code up on GitHub)

After that, this PR will automatically update with the updated code. 

@nai-aung26 I believe you have the same problem so you can follow these steps.

Reply:
Hi Brenton, I try to fix manually, can you check it please?